changelog/feature-add_ian_for_et

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: feat
+
+Add In-App Notifications for Event Tickets [ET-2294]
```

Comment:
This should be something different. "Use the hook in TCMN to add evnet settings pages" or such